images/locals.tf

```diff
@@ -4,4 +4,19 @@ locals {
     "tcp",
     "udp",
   ]
+
+  # Get current Images account ID from Images provider
+  this_account_id = data.aws_caller_identity.images.account_id
+
+  # Look up current Images account name from AWS organizations provider
+  this_account_name = [
+    for account in data.aws_organizations_organization.cool.accounts :
+    account.name
+    if account.id == local.this_account_id
+  ][0]
+
+  # Determine Images account type based on AWS account name
+  # Account name format:  "ACCOUNT_NAME (ACCOUNT_TYPE)"
+  #         For example:  "Images (Production)"
+  this_account_type = length(regexall("\\((.*?)\\)", local.this_account_name)) == 1 ? regex("\\((.*?)\\)", local.this_account_name)[0] : "Unknown"
```

Comment:
I would write this regex as `\\(([~()]*)\\)`.  It's not clear to me what the `?`s are doing.  Do they make the match optional, so `regexall` doesn't error out?

Reply:
No, the `regexall` should never error out- the worst it can do is return an empty list `[]`.  I was going for a [lazy quantifier solution](https://www.rexegg.com/regex-quantifiers.html#lazy_solution), but I have no issue using the regex you wrote.  Regexes have never been my strong suit, so I'll defer to you here.

Reply:
I see.  I think with the regex I specified you don't need the lazy quantifier, since `()` are excluded from the match.

Trust but verify!

Reply:
Changed in https://github.com/cisagov/cool-accounts/pull/42/commits/6c3e9751ec6534022dfc772b3107f85d6c537a0c.